specs/player_spec.rb

```diff
@@ -0,0 +1,163 @@
+require_relative 'spec_helper'
+
+describe "Player" do
+
+  before do
+    @test_ob = Scrabble::Player.new("name")
+    @test_tile_bag = Scrabble::Tilebag.new
+  end
+
+  describe "initialize method" do
+    it "Is a Player object" do
+      @test_ob.must_be_instance_of Scrabble::Player
+    end
+
+    it "Can be instantiated with one argument" do
+      @test_ob.name.must_equal "name"
+    end
+
+    it "Has an instance variable @plays, which is an array" do
+      @test_ob.plays.must_be_instance_of Array
+    end
+
+    it "Has an instrance variable @total_score, which is an Integer" do
+      @test_ob.total_score.must_be_instance_of Integer
+    end
+
+    it "Has an instance variable @total_score, that returns the current total score" do
+
+      test_players_tiles = @test_ob.draw_tiles(@test_tile_bag)
+      test_word = test_players_tiles[0]
+      @test_ob.play(test_word)
+
+      score_test_word = Scrabble::Scoring.score(test_word)
+
+      @test_ob.total_score.must_equal score_test_word
+    end
+
+  end#initialize
+
+  describe "play method" do
+    it "Can be called" do
+      @test_ob.must_respond_to :play
+    end
+
+    it "Raises an ArgumentError if input is not a String" do
+      proc { @test_ob.play(123)}.must_raise ArgumentError
+    end
+
+    it "Adds the word input to the @plays instance variable" do
+      test_players_tiles = @test_ob.draw_tiles(@test_tile_bag)
+      test_word = test_players_tiles[0]
+
+      @test_ob.play(test_word)
+      @test_ob.plays.must_equal [test_word]
+    end
+
+    it "Returns false if player has already won" do
+
+      2.times do
```

Comment:
Good test case! This is an important behavior for the game, but it would be an easy one to miss.